x-pack/plugins/enterprise_search/public/applications/app_search/index.tsx

```diff
@@ -63,7 +63,7 @@ export const AppSearchConfigured: React.FC<IInitialAppData> = (props) => {
         <SetupGuide />
       </Route>
       <Route>
-        <Layout navigation={<AppSearchNav />}>
+        <Layout navigation={<AppSearchNav />} readOnlyMode={readOnlyMode}>
```

Comment:
I debated passing `readOnlyMode` as a prop vs calling it directly in Layout via `const { readOnlyMode } = useValues(HttpLogic)`, but eventually went with passing a prop since our `index` files already are already hooking into HttpLogic 🤷 I could go either way though if someone else feels strongly about this!